src/Engine/ProtoCore/Parser/Parser.cs

```diff
@@ -176,6 +176,66 @@ private bool IsIdentList()
         return false;
     }
 
+    private bool IsListExpression()
```

Comment:
I don't see this method in the ATG?

Reply:
Good catch, @aparajit-pratap! I deleted that at the last minute from the atg as it was unused. I'll regen.